### PR TITLE
SPaG fixes

### PR DESCRIPTION
**Pull request by CoffeeBrewer64**
the things I changed just stuck out to me, I had to do it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-Vita3K Android is an experimental PlayStation Vita emulator for Android. It comes as an APK.
+Vita3K Android is an experimental PlayStation Vita emulator for Android. It comes as an APK, ready for you to install.
 
-Minimum requirements are as followed:
-- ARM64 device with
+Minimum requirements are as follow:
+- ARM64 device
 - Vulkan 1.0 support
-- Android 7+ (Note: if your device supports Vulkan 1.0, then it is running at least Android 7)
+- Android 7+ (Note: if your device supports Vulkan 1.0, then it is running at least Android 7, and vice versa)
```
Comment (Macdu) [inline README.md:9]: I'm not sure all the android 7 devices support vulkan

Comment (CoffeeBrewer64) [reply README.md]: fixed, also added some other changes

**Comment by Raulzitoml743** [comment]
Macdu, good afternoon, my name is Andre, I´m from Brazil and the first thing is thank you for the release of this incredible emulator, I have one thing to report, the game YS origins simply don´t save, everytime I try the game crashes, is there something that can be done to solve that right now or it´s a problem of the version of the emulator itself, thank you again and sorry for the incovenience

**Comment by CoffeeBrewer64** [comment]
you can message on the discord server, please don't hook onto my pr like that

**Comment by Raulzitoml743** [comment]
> you can message on the discord server, please don't hook onto my pr like that

ok sorry, deleting message

**Pull request merged**